### PR TITLE
Fix Broken gerrit support for autobumper

### PR DESCRIPTION
**Pull request by mpherman2**
This PR Re-adds setting http cookiefile and adding upstream. It also changes pushing to upstream, and manually sets username and email for git user. We also changed the commit message to hide the identifier.

/assign @chaodaiG

### CODE REVIEW

File: prow/cmd/generic-autobumper/bumper/bumper.go [hunk 1]
```diff
@@ -1004,7 +1006,19 @@ func GerritNoOpChange(changeID, hostRepo string) (bool, error) {
 
 }
 
-func createCR(msg, branch, changeID, hostRepo, cookieFile string, reviewers, cc []string, stdout, stderr io.Writer) error {
+func createCR(msg, branch, changeID, hostRepo, cookieFile, author, email string, reviewers, cc []string, stdout, stderr io.Writer) error {
+	if err := Call(stdout, stderr, gitCmd, "config", "http.cookiefile", cookieFile); err != nil {
+  		return fmt.Errorf("unable to load cookiefile: %v", err)
+  }
```
Comment (chaodaiG) [inline prow/cmd/generic-autobumper/bumper/bumper.go:1012]: indentation doesn't seem right

File: prow/cmd/generic-autobumper/bumper/bumper.go [hunk 2]
```diff
@@ -1004,7 +1006,19 @@ func GerritNoOpChange(changeID, hostRepo string) (bool, error) {
 
 }
 
-func createCR(msg, branch, changeID, hostRepo, cookieFile string, reviewers, cc []string, stdout, stderr io.Writer) error {
+func createCR(msg, branch, changeID, hostRepo, cookieFile, author, email string, reviewers, cc []string, stdout, stderr io.Writer) error {
+	if err := Call(stdout, stderr, gitCmd, "config", "http.cookiefile", cookieFile); err != nil {
+  		return fmt.Errorf("unable to load cookiefile: %v", err)
```
Comment (chaodaiG) [inline prow/cmd/generic-autobumper/bumper/bumper.go:1011]: Can we also include `stdout` as part of error message?

Comment (mpherman2) [reply prow/cmd/generic-autobumper/bumper/bumper.go]: No need here. stdout is already output.

File: prow/cmd/generic-autobumper/bumper/bumper.go (hunk 1 above)
Comment (mpherman2) [reply prow/cmd/generic-autobumper/bumper/bumper.go]: Done. Very unhappy with my new temporary IDE :'(

**Review by chaodaiG — APPROVED** [review]

**Pull request merged**